source/includes/fact-mongoshrc-loading.rst

```diff
@@ -1,24 +1,20 @@
-On startup, :binary:`mongosh` checks the user's ``HOME`` directory for
-a JavaScript file named :ref:`.mongoshrc.js <mongoshrc-js>`. If this
-file is found, :binary:`mongosh` reads the content of
-:ref:`.mongoshrc.js <mongoshrc-js>` before displaying the prompt for
-the first time.
+On startup, :binary:`mongosh` checks your ``HOME`` directory for a
```

Comment:
Should .mongoshrc.js be enclosed in backticks, ``.mongoshrc.js`` ?

Reply:
It was like this originally, but this page looked really bad with the monospace styling for mongoshrc.js. The frequency at which we use the term made the page really hard to read with constant switching of styles, so I opted for plaintext.

Reply:
FWIW - Our styling is way too busy, I wish we had more say in how the docs look.